src/DatePicker/Utilities.elm

```diff
@@ -543,3 +532,24 @@ doDaysMatch zone dateTimeOne dateTimeTwo =
             Time.posixToParts zone dateTimeTwo
     in
     oneParts.day == twoParts.day && oneParts.month == twoParts.month && oneParts.year == twoParts.year
+
+
+calculatePad : Weekday -> Weekday -> Bool -> Int
+calculatePad firstWeekDay monthStartDay isFrontPad =
+    let
+        reversedListOfWeekDay =
```

Comment:
Why is this called `reversedListOfWeekDay`?

Reply:
Ah, I fixed that. It was reversed before...